src/app/popup/popup.css

```diff
@@ -127,5 +127,6 @@ p {
 @media (max-width: 425px) {
   .panel {
     width: auto !important;
+    max-width: 425px;
```

Comment:
@Jackymancs4 - I added this to prevent the popup from stretching too wide when not in the overflow window. It seems to work, but does it seem reasonable to you?

Reply:
Before:
![image](https://user-images.githubusercontent.com/38854/53371028-25bb3d80-3947-11e9-8d58-57b642443ffd.png)



After:
![image](https://user-images.githubusercontent.com/38854/53371092-42f00c00-3947-11e9-88a3-65503bc2106e.png)

Reply:
Actually, do these styles need to be responsive at all? What about just using the following in the .panel class?
```
panel {
  ...
  min-width: 20em;
  max-width: 425px;
  ...
```


* If in the overflow menu, the width is fixed. 
* If not in the overflow menu, the width will adjust from 20em to 425px.

Reply:
BTW: what was the reason for using 425px in the @media query? I wonder if the actual width when in the overflow menu is DPI-dependent.

Reply:
> It seems to work, but does it seem reasonable to you?

That looks perfect! Wider but not too wide.

> Actually, do these styles need to be responsive at all?

After quite a few of experiments, I found that the answer is no. The whole responsiveness issue was fixed when I changed `width: 20em;` to `min-width: 20em;` thanks to the behavior of `display: flex; flex-direction: column;`.
You can safely delete the media-query.

> BTW: what was the reason for using 425px in the @media query?

I found that value [here](https://discourse.mozilla.org/t/can-add-ons-webextensions-popups-determinate-whether-they-are-shown-in-the-overflow-menu-or-not/27937/5). The guy in the comment claims it to be a fixed value and I effectively used that trick in other addons.

> I wonder if the actual width when in the overflow menu is DPI-dependent.

After a lot more experiments, the width varied from 360px to 400px on monitor with different resolution and density. After this I feel there is a little teeny-tiny chance that on ultra wide >8k monitors the overflow menu could be wider than 425px. No certainties. Still
```
panel {
  ...
  min-width: 20em;
  max-width: 425px;
  ...
```
is what I call a sane default and the result is definitely nice!

Reply:
That's great, many thanks for your help. I'll update this PR with those suggestions.